### DISCUSSION TITLE
Custom reporting for API

### DISCUSSION THREAD

**Discussion by scottmando2000** [plausible/analytics#974] [Anything Else]
Would there be a way I would be able to custom report data from my server for API requests? Since no JS would be ran for a JSON response I'm curious if this is possible.

I didn't see anything in the documentation about it but found `POST /api/event` in the router as well as in the hosted JS.

**Comment by Vigasaurus** [comment]
https://github.com/plausible/analytics/discussions/234#discussioncomment-382572 Yep! There's a lot of discussion + some ideas on how to do it here.

**Reply by scottmando2000** [reply]
Thanks for pointing me in the right direction! Might have to take some of the info from there and make a package out of it.

**Comment by metmarkosaric** [comment]
we've now documented how to do this here https://plausible.io/docs/events-api
